data/strings/strings.txt

```diff
@@ -15613,7 +15615,7 @@
 
   [error_enter_correct_twitter_page]
     tags = android
-    en = Enter a valid Twitter web address or username
+    en = Enter a valid Twitter username or URL
```

Comment:
As above, but I'll stop now since I wanted to say it once but I don't want to just be annoying.